Woof/Woof/Model/Walking/Walking+Dummy.swift

```diff
@@ -0,0 +1,75 @@
+import Foundation
+
+extension Walking {
+    /// Provides dummy data for `Walking` model.
+    enum Dummy {
+        static let dummyWalking = Walking(
+            id: UUID(),
+            owner: Owner.Dummy.laraCroft,
+            sitter: Sitter.Dummy.emilyDoe,
+            status: .available,
+            start: Date(),
+            end: Date() + 2400,
+            location: Location(longitude: 0.0, latitude: 0.0, address: "Sample Address"),
+            reviews: Reviews(ownerReview: "", sitterReview: ""),
+            rating: Ratings(ownerRating: 0, sitterRating: 0),
+            notes: "Sample notes",
+            price: 28.0 // You can set the price as needed
+        )
+
+        static let bulkDummyWalkings = [
+            Walking(
+                id: UUID(),
+                owner: nil,
+                sitter: Sitter(), // You may want to create a Sitter instance here
+                status: .finished, // You can change the status as needed
```

Comment:
Do we need these comments?
Please review the rest of the code.

Reply:
Perhaps they can be useful for a person who wants to change something.
Fixed with 7f70e76